MeetupPlatformApi/Domain/Meetup.cs

```diff
@@ -11,4 +13,8 @@ public class Meetup
     public DateTime EndTime { get; set; }
 
     public string Description { get; set; }
+
+    public Guid OrganizerId { get; set; }
+
+    public List<PlainUser> Users { get; set; }
```

Comment:
Is it really necessary to change the metup domain model ? Why not create a table with a many-to-many relationship, where you can store the ID of the meetup and the ID of the user who subscribed to it. In this case, you do not have to chamge meetup domain model.

Reply:
My fault, you'll have to change the model anyway.